pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -38,8 +38,11 @@ def __init__(
         self.working_dir = Path(working_dir).absolute()
         self.dry_run = dry_run
         self.release_version = tuple(map(int, self.release_name.split(".", 2)))
+        self.target_release = (
```

Comment:
`target_release` is a concept only valid to bugzilla. would like to keep this a local variable rather than a class field since the name is confusing.